clientGQL_test.go

```diff
@@ -128,7 +132,18 @@ func (testDataTemplater *TestDataTemplater) ParseTemplatedString(contents string
 	return strings.TrimSpace(data.String())
 }
 
+func stripWhitespace(input string) string {
```

Comment:
Adding this ensures that we can have readable templates - see https://github.com/OpsLevel/opslevel-go/pull/483/commits/7fb2cddcbd5495177af4e3a899fb83849a81d185